android/src/main/java/com/swmansion/reanimated/nodes/PropsNode.java

```diff
@@ -89,12 +89,32 @@ protected Double evaluate() {
         }
       } else {
         String key = entry.getKey();
+        Object value = node.value();
+        WritableMap dest;
+
         if (mNodesManager.uiProps.contains(key)) {
           hasUIProps = true;
-          mPropMap.putDouble(key, node.doubleValue());
-        } else {
+          dest = mPropMap;
```

Comment:
This check appears to be unnecessary, because we wish to update this properties only via jsBridge.

Reply:
I was under the impression that `nativeProps` refers to the props that can be successfully updated with `setNativeProps` from js thread. Not sure how to test that, adding `text` with `addWhitelistedNativeProps` and updating TextInput doesn’t crash (you can check example from this pr). Previously it wasn’t expecting that you can even have jsProps here so maybe I was wrong to change that. I’ll need some time to test that (probably over the weekend).

Reply:
Ok, however I believe be could lease as normal `jsProps` because actually I'm not very sure if it won't break anything and and it's very profitable IMHO.